docs/en/advanced-concepts/migrating-to-xcuitest.md

```diff
@@ -1,4 +1,4 @@
-## Migrating your iOS tests from UIAutomation (iOS 9.2 and below) to XCUITest (iOS 9.3 and up)
+## Migrating your iOS tests from UIAutomation (iOS 9.3 and below) to XCUITest (iOS 9.3 and up)
```

Comment:
Has Apple fixed the location end-point issue for XCUITest, yet?  Would like to take a look at the Apple request.